lms/static/sass/views/_teams.scss

```diff
@@ -83,41 +84,56 @@
 
   .page-header-search {
 
+    .wrapper-search-input {
+      position: relative;
+      vertical-align: middle;
+      display: inline-block;
+    }
+
     .search-label {
         @extend %text-sr;
     }
 
     .search-field {
         transition: all $tmg-f2 ease-in-out;
-        border: 0;
-        border-bottom: 2px solid transparent;
+        border: 1px solid $gray-l4;
+        border-radius: 3px;
         padding: ($baseline/4) ($baseline/2);
         font-family: inherit;
         color: $gray;
-        @include text-align(right);
-
-        &:focus {
-          border-bottom: 2px solid $black;
-          color: $black;
-        }
     }
 
     .action-search {
       @extend %button-reset;
-      padding: ($baseline/4) ($baseline/2);
-
+      padding: ($baseline/5) ($baseline/2);
+      vertical-align: middle;
+      background-color: $gray-l3;
+      text-shadow: none;
 
       .icon {
-          color: $gray-l3;
+          color: $white;
       }
 
       // STATE: hover and focus
       &:hover,
       &:focus {
+        background-color: $blue;
+      }
+    }
 
-        .icon {
-            color: $black;
-        }
+    .action-clear {
+      @extend %button-reset;
+      position: absolute;
+      top: 0;
+      right: 0;
```

Comment:
Thanks @marcotuts. That means that the line should move to the top as includes go first, yes?